BucketeerTests/EventInteractorTests.swift

```diff
@@ -571,4 +571,4 @@ final class EventInteractorTests: XCTestCase {
         wait(for: [expectation], timeout: 1)
     }
 }
-// swiftlint:enable [type_body_length file_length]
+// swiftlint:enable type_body_length file_length
```

Comment:
Just confirming. Does it need to enable at the end of the file?

Reply:
yes, we need to re-enable it at the end of the file, otherwise we will have one more warning like that 

<img width="1037" alt="Screenshot 2023-06-19 at 10 46 24" src="https://github.com/bucketeer-io/ios-client-sdk/assets/2597710/67cf4d38-b04f-4136-9e23-40144798ff65">